### PR TITLE
TASK #2075 | Add user settings placeholder

### LINKED ISSUE #2075: Placeholder for input fields should be configurable in user settings dialog
It should be possible to specify placeholder values for input fields in user settings config, like:
```
userAccount: {
      label: 'User Account',
      sublabel: 'username',
      icon: 'account',
      title: 'User Account',
      settings: {
        name: { type: 'string', label: 'Name', placeholder: 'Type your name' },
        ...
        }
 }
```
It should be optional.

### PR DESCRIPTION
**Pull request by rafalgamon**
**Description**

Changes proposed in this pull request:

- #2075 Placeholder for input fields should be configurable in user settings dialog

**Related issue(s)**
`Fixes #2075`

### CODE REVIEW

File: core/src/UserSettingsEditor.html [hunk 1]
```diff
@@ -155,6 +155,7 @@
             class="fd-input"
             type="text"
             aria-label="Image label"
+            placeholder="{schemaItem.placeholder && schemaItem.placeholder}"
```
Comment (JohannesDoberer) [inline core/src/UserSettingsEditor.html:158]: Why do you check it twice?

File: core/src/UserSettingsEditor.html [hunk 2]
```diff
@@ -180,6 +181,7 @@
                   <input
                     type="text"
                     class="fd-input fd-input-group__input"
+                    placeholder="{schemaItem.placeholder && schemaItem.placeholder}"
```
Comment (JohannesDoberer) [inline core/src/UserSettingsEditor.html:184]: Why do you check it twice?

**Review by JohannesDoberer — COMMENTED** [review]
Could you also add an e2e test?

File: core/src/UserSettingsEditor.html (hunk 1 above)
Comment (rafalgamon) [reply core/src/UserSettingsEditor.html:158]: Cause first one check is is exist and second one add value

File: core/src/UserSettingsEditor.html (hunk 2 above)
Comment (rafalgamon) [reply core/src/UserSettingsEditor.html:184]: Cause first one check is is exist and second one add value

File: docs/user-settings.md [hunk 3]
```diff
@@ -125,6 +125,7 @@ These parameters used in the example above allow you to configure the items in t
 
   - **label** (optional) is a string and the label of the setting.
   - **isEditable** (optional) is a boolean and by default `true`. If it is set to `false` the setting is not editable.
+  - *placeholder** (optional) is a string and by default `null`. If it is set to `Value` the placeholder is displaing `Value` the other hend notdsda.
```
Comment (JohannesDoberer) [inline docs/user-settings.md:128]:
```suggestion
  - **placeholder** (optional) is a string and by default `null`. If it is set to `value` the placeholder is displaying the `value` the other hend notdsda.
```
What do you mean at the end of the sentence?

**Review by JohannesDoberer — APPROVED** [review]
LGTM  👍 !

**Pull request merged**